tests/Agent/UnitTests/CompositeTests/CrossAgentTests/DistributedTracing/TraceContextCrossAgentTests.cs

```diff
@@ -16,6 +15,7 @@
 using System.IO;
 using System.Linq;
 using System.Reflection;
+using MoreLinq;
```

Comment:
This change seems to have led to a failed test.

Reply:
It did, but the test was using a `.ForEach` method from `Castle.Core.Internal` that it almost certainly shouldn't have been using in the first place. I'll either revert the change or find a way to make the test work.